Setup/UpgradeData.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
```

Comment:
@NandhiniKarthi These should be replaced with Data patches in favor of declarative schema